src/main/java/pl/kulinarnie/demo/CategoryRepository.java

```diff
@@ -0,0 +1,10 @@
+package pl.kulinarnie.demo;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+public interface CategoryRepository extends JpaRepository<Category, Long> {
+
+    List<Category> findAll();
```

Comment:
Jeżeli dziedziczysz po JpaRepository, to tam już jest metoda findAll, nie ma potrzeby jej zpaisywania tutaj